app/views/lab_spaces/show.html.erb

```diff
@@ -68,7 +68,7 @@
               </div>
             </div>
         <% end %>
-        <% if user_signed_in? %>
+        <% if policy(:equipment).new? %>
```

Comment:
Did this policy already exist? Or is it new?

Reply:
Yes, it already exist